beets/ui/__init__.py

```diff
@@ -1060,6 +1060,32 @@ def parse_subcommand(self, args):
         suboptions, subargs = subcommand.parse_args(args)
         return subcommand, suboptions, subargs
 
+    @staticmethod
+    def _store_dict(option, opt_str, value, parser):
```

Comment:
This is a nice way to parse a series of `foo=bar` options! Cool!

But I don't think this needs to be a static method of the `SubcommandsOptionParser` class, which is intended for a pretty specific purpose. Can we make it a plain, top-level function?